instat/dlgMosaicPlot.vb

```diff
@@ -225,7 +225,7 @@ Public Class dlgMosaicPlot
     End Sub
 
     Private Sub cmdMosaicPlotOptions_Click(sender As Object, e As EventArgs) Handles cmdMosaicPlotOptions.Click
-        sdgLayerOptions.SetupLayer(clsNewGgPlot:=clsGgplotFunction, clsNewGeomFunc:=clsMosaicGeomFunction, clsNewGlobalAesFunc:=clsAesFunction, clsNewLocalAes:=clsLocalAesFunction, bFixGeom:=True, ucrNewBaseSelector:=ucrSelectorMosaicPlot, bApplyAesGlobally:=True, bReset:=bResetBoxLayerSubdialog)
+        sdgLayerOptions.SetupLayer(clsNewGgPlot:=clsGgplotFunction, clsNewGeomFunc:=clsMosaicGeomFunction, clsNewGlobalAesFunc:=clsAesFunction, clsNewLocalAes:=clsLocalAesFunction, bFixGeom:=True, ucrNewBaseSelector:=ucrSelectorMosaicPlot, bApplyAesGlobally:=False, bReset:=bResetBoxLayerSubdialog)
         sdgLayerOptions.ShowDialog()
```

Comment:
@dannyparsons I changed this to `False` because we did not want the apply on all layers chaecked on the sdglayers because that would put all the aes on the ggplot function and we want them on the geom mosaic instead

Reply:
We should do SetRCode for everything after coming back from subdialogs I think

Reply:
yeah we can do that. But I am not sure how this would work with the multiple receiver eg- When we have multiple receiver on the main dlg and we only have single receiver on layer options

Reply:
`x` and `fill` don't work in the same way for this geom because of using `product()` but that's fine as long as they are not changed on layer options.

Reply:
So should we make it that they are non editable on the layers options?

Reply:
if we have an easy way to just make x and fill non editable then yes but I didn't think we have that

Reply:
So it seems that it removes some variables if there was more than one on the main dialog for `x` or `conds`. So we need to do something to prevent that.
Actually `fill` is ok because that's a single receiver.

Reply:
There another issue that the subdialog changes the geom from `ggmosaic::geom_mosaic` to `ggplot2::geom_mosaic`. We might want to disable the aes tab completely for now.